tutorial/register-oracle.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "inventory-connector",
+    "config": {
+        "connector.class" : "io.debezium.connector.oracle.OracleConnector",
+        "tasks.max" : "1",
+        "database.server.name" : "server1",
+        "database.hostname" : "<docker host ip>",
+        "database.port" : "1521",
+        "database.user" : "c##xstrmadmin",
+        "database.password" : "xsa",
+        "database.dbname" : "ORCLCDB",
+        "database.pdb.name" : "ORCLPDB1",
+        "database.out.server.name" : "XOUT",
```

Comment:
This should be the same as the one defined in https://github.com/debezium/oracle-vagrant-box/blob/master/README.md#setting-up-oracle-db